packages/babel-traverse/src/scope/index.js

```diff
@@ -535,13 +535,18 @@ export default class Scope {
 
         parent.references[name] = true;
 
-        this.bindings[name] = new Binding({
-          identifier: id,
-          existing: local,
-          scope: this,
-          path: bindingPath,
-          kind: kind,
-        });
+        // A redeclaration of an existing variable is a modification
+        if (name in this.bindings) {
```

Comment:
Use `#hasOwnBinding`, please.

Reply:
Actually, just check `if (existing)`

Reply:
`if(this.hasOwnBinding(name))` works; can't do second because no `existing` is in scope. How do I proceed?

Reply:
Sorry, it's `local`.

Reply:
Check, works. Pushed new version.